src/components/sbb-breadcrumb/sbb-breadcrumb.tsx

```diff
@@ -0,0 +1,113 @@
+import { Component, ComponentInterface, Element, h, Host, JSX, Prop, State } from '@stencil/core';
+import {
+  LinkProperties,
+  LinkTargetType,
+  resolveLinkOrStaticRenderVariables,
+  targetsNewWindow,
+} from '../../global/interfaces/link-button-properties';
+import { i18nTargetOpensInNewWindow } from '../../global/i18n';
+import {
+  actionElementHandlerAspect,
+  createNamedSlotState,
+  documentLanguage,
+  HandlerRepository,
+  languageChangeHandlerAspect,
+  namedSlotChangeHandlerAspect,
+} from '../../global/helpers';
+
+/**
+ * @slot unnamed - Use this to slot the breadcrumb's text.
+ * @slot icon - Use this to display an icon as breadcrumb.
+ */
+@Component({
+  shadow: true,
+  styleUrl: 'sbb-breadcrumb.scss',
+  tag: 'sbb-breadcrumb',
+})
+export class SbbBreadcrumb implements ComponentInterface, LinkProperties {
+  /** The href value you want to link to. */
+  @Prop() public href: string | undefined;
+
+  /** Where to display the linked URL. */
+  @Prop() public target?: LinkTargetType | string | undefined;
+
+  /** The relationship of the linked URL as space-separated link types. */
+  @Prop() public rel?: string | undefined;
+
+  /** Whether the browser will show the download dialog on click. */
+  @Prop() public download?: boolean;
+
+  /**
+   * The icon name we want to use, choose from the small icon variants
+   * from the ui-icons category from here
+   * https://lyne.sbb.ch/tokens/icons/.
+   * Inline variant doesn't support icons.
+   */
+  @Prop() public iconName?: string;
+
+  /** Sets the aria-current attribute. */
+  @Prop() public ariaCurrent: string | undefined;
```

Comment:
It's only for internal use as it is set by the group. Prefix to not provide somehow two times aria-current (on host and link)
```suggestion
  /** 
  /* @internal
  /* Sets the aria-current attribute. 
  */
  @Prop() public accessibilityAriaCurrent: string | undefined;
```

Reply:
I double checked with Voiceover and I found that it's not read as expected; I moved it to the host and now it works (commit eb3f825a).

Reply:
Ah yes! because the role is on the host as well...

Reply:
@DavideMininni-Fincons you can now completely remove the prop

Reply:
Done in 3baad7dc ;)